l1-contracts/src/mock/StakingAssetHandler.sol

```diff
@@ -153,6 +175,7 @@ contract StakingAssetHandler is IStakingAssetHandler, Ownable {
     } else {
       _topUpIfRequired(depositAmount);
 
+      _validateMerkleProof(_attester, _merkleProof);
```

Comment:
A note somewhere about what is supposed to be in the depositMerkleRoot would make it slightly easier to read through.